server/manifest.go

```diff
@@ -22,8 +22,8 @@ const manifestStr = `
   "min_server_version": "5.12.0",
   "server": {
     "executables": {
-      "linux-amd64": "server/dist/plugin-linux-amd64",
       "darwin-amd64": "server/dist/plugin-darwin-amd64",
+      "linux-amd64": "server/dist/plugin-linux-amd64",
```

Comment:
Not sure why this got changed with `make apply`, but the linter was complaining that I had to run it :shrug: